src/membership_check.rs

```diff
@@ -6,6 +6,12 @@ use tide::{Response, StatusCode};
 // The info! logging macro comes from crate::azure_function::logger
 use crate::azure_function::{AzureFnLogger, AzureFnLoggerExt};
 use crate::{AppRequest, MailchimpQuery, MailchimpResponse};
+use chrono::{NaiveDate, ParseError, Local, TimeZone};
+use chrono_tz::America::Vancouver;
+
+fn parse_iso_date(iso_date: &str) -> Result<NaiveDate, ParseError> {
+    NaiveDate::parse_from_str(iso_date, "%Y-%m-%d")
+}
```

Comment:
@glindstr I see the existing code in the ipn handler does something like this but it probably isn't ideal.

better would be to use [`DateTime::parse_from_rfc3339`](https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.parse_from_rfc33390) _(RFC 3339 is a subset of ISO 8601)_ 

The [relevant code in the ipn handler](https://github.com/squamishaccess/squamishaccess-functions/blob/328ea1d90cf805083fd27f07fe9ac7599c4478b9/src/ipn_handler.rs#L257-L259) should probably also use [`DateTime::with_time`](https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.with_time) as per the example for that, rather than how is is currently adding 365 days.

Reply:
I can address this. Depending on our the choice to use NaiveDate type it doesn't have a rfc3338 parsing method.